configuration/configurator/schemas/SchemaAttributes.cpp

```diff
@@ -105,6 +105,10 @@ void CAttribute::getDocumentation(StringBuffer &strDoc) const
             return; // HIDDEN
         else
             pToolTip = pAppInfo->getToolTip();
+
```

Comment:
I just follows his previous code. Guess he want to prevent the string changed and also avoid buffer copy.